integration_test/index_test.go

```diff
@@ -84,3 +84,50 @@ func TestKrewIndexList_NoIndexes(t *testing.T) {
 		t.Fatalf("expected index list to be empty:\n%s", string(out))
 	}
 }
+
+func TestKrewIndexRemove_nonExisting(t *testing.T) {
+	skipShort(t)
+	test, cleanup := NewTest(t)
+	test = test.WithEnv(constants.EnableMultiIndexSwitch, 1)
+	defer cleanup()
+
+	err := test.Krew("index", "remove", "non-existing").Run()
+	if err == nil {
+		t.Fatal("expected error")
+	}
+}
+
+func TestKrewIndexRemove_ok(t *testing.T) {
+	skipShort(t)
+	test, cleanup := NewTest(t)
+	test = test.WithEnv(constants.EnableMultiIndexSwitch, 1)
+	defer cleanup()
+
+	test.Krew("index", "add", "foo", constants.IndexURI).RunOrFail()
```

Comment:
That does another clone of the remote repo. Can we avoid that by using a local clone?

Reply:
Will do.

Reply:
Done.